package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
In what part of this file would a task to fire up the server be added if we wanted to?